make_release/get_coverage.nu

```diff
@@ -0,0 +1,31 @@
+# Test coverage gathering for nushell 
+# Uses cargo-llvm-cov
+# Uses separate execution to track the integration tests
+# Hacked together by @sholderbach
+
+# Get test coverage for nushell
+def main [
+       --extra # Get coverage for extra features
+   ] {
+   cargo llvm-cov show-env --export-prefix | 
+       lines | 
+       str substring '7,' | 
+       split column '=' | 
+       str trim -c '"'  | 
+       transpose | 
+       headers | 
+       reject 'column1' | 
+       get 0 | 
```

Comment:
I'm not sure it's necessary but it may be worth throwing in a `| str trim` after `get 0` because I noticed there are leading spaces in the RUSTFLAGS env var.

Reply:
```suggestion
       get 0 | 
       str trim |
```